Adam Overfitting Pull Req

adpoleski1 opened the pull request.

adpoleski1 committed:
Combined UI and server

adpoleski1 committed:
minor edits to app to fit style guide

adpoleski1 committed:
updated prereq page

adpoleski1 committed:
button editing and other minor edits

adpoleski1 committed:
minor edits to grammar

lbednarczyk commented on app.R:
```diff
@@ -0,0 +1,472 @@
+    skin = "black",
+    dashboardHeader(
+      title = "Overfitting",
+      titleWidth = 235, 
```
The title width should be 250 instead of 235.

lbednarczyk commented on app.R:
```diff
@@ -0,0 +1,472 @@
+    
+    #Sidebar
+    dashboardSidebar(
+      width = 235,
```
The width here should be 250 too.

lbednarczyk commented on app.R:
```diff
@@ -0,0 +1,472 @@
+                    size = "large", 
+                    style = "default")),
+                br(),
+                h3(strong("Acknowledgements")),
```
I think that the bold words should be changed to just normal text.

lbednarczyk commented on app.R:
```diff
@@ -0,0 +1,472 @@
+                  many choices.  This is just one of several different types
+                  of overfitting that can occur."),
+                br(),
+                tags$ul(
```
The tags$ul makes the instructions header and bullets not aligned with the rest of the headers.

lbednarczyk commented on app.R:
```diff
@@ -0,0 +1,472 @@
+                h1("Understanding the Overfitting Effect"),
+                withMathJax(),
+                tags$ul(
+                tags$li("A reasearcher looks at many explanatory variables
```
There's a typo in researcher.

lbednarczyk commented on app.R:
```diff
@@ -0,0 +1,472 @@
+                
+                tags$li("But if we draw another sample randomly from the 
+                                      same model, it will not fit nearly as well.")),
+                br(),
```
I think it would look better if one of the breaks were taken out, the layout looks a little awkward with the large space.

lbednarczyk commented on app.R:
```diff
@@ -0,0 +1,472 @@
+                     variables, the sample size, and the population correlation between the 
+                     explanatory variables and the response variable. Later, the 
+                     researcher will run a validation study with new, independent
+                     observations for X."),
```
I'm not sure if the maroon background would align with the style guide, maybe a well panel would be a better option.

lbednarczyk commented on app.R:
```diff
@@ -0,0 +1,472 @@
+    kk = which.max(abs(R2))
+    mm <- lm(y2~xmat[,kk]) # Best Chosen X
+    
+    plot(xmat[,kk], y2, xlab = "Best Chosen X", ylab = "Y",font.lab = 2, cex = 1.5)
```
Two of the plots have axis labels in bold and the other isn't bold, they should all be the same.

lbednarczyk commented on app.R:
```diff
@@ -0,0 +1,471 @@
+                     sliderInput("n", "Sample Size:", min = 2, max = 50, 
+                                 value = 5 ,
+                                 step = 1),
+                     bsPopover("n", "", "Number of Observations", 
+                               place = "right",
+                               options = list(container = "body")),
```
I know that we're supposed to limit the use of popovers and I think a lot of these ones aren't really necessary and could be removed.

lbednarczyk commented on app.R:
```diff
@@ -0,0 +1,471 @@
+                     observations for X."),
+
+                     
+                     sliderInput("n", "Sample Size:", min = 2, max = 50, 
```
A lot of these functions should be made to be explicit, like adding inputId= and label=.

lbednarczyk commented on app.R:
```diff
@@ -0,0 +1,471 @@
+                  h3("Plot:"),
+                  column(4,   
+                         align = "center",
+                         plotOutput("scatter", height = 360),
+                         tableOutput("choose"),
```
On the explore page, the top two plots aren't in line with each other and it would look better if they were.

lbednarczyk reviewed (dismissed):
Hi Adam,
The app works great overall and I didn't find any bugs. The only issues I found were with formatting, which I left comments for throughout the code. Also, the code itself could be made more explicit and some spacing could be fixed.

JuddHe reviewed (dismissed):
1： the app can run.
2: there are two issues in the Github issue, and both of them haven't been addressed.
3: 
-  the font of the whole app is not right (too small compared to other apps).
-  in the right corner of the overview page, there should be three icons over there. In this app, the three icons do not exist. I do see the codes about these three icons, but they might be wrong.
-  the order of the pages might be wrong. Overall might be the first page.
-  no reference page.
- for the acknowledge part in the overview page, the contents in the app are different from the contents in the code p("....").
- there is a word type. wrong spelling for the word "researcher".
-  in explore page, the two plots does not in the same line.
-  for the first plot, the two-axis were labeled in bold, but the two-axis were not labeled in bold in the second plot.

Recommended Priority: High

JuddHe commented:
3: 
-  no description file.

JuddHe committed:
changes on files and code

delete challenge in explore page, and change challenge part to contents in prerequisites page .
add description file and screenshot.

JuddHe committed:
Update README file

JuddHe committed:
Update

JuddHe committed:
small update

JuddHe committed:
update app.R

line two graphs with each other. (in explore page)

ymy5178 reviewed (commented):
Recommended Priority: Urgent

There are just two small format things.

1. In the Overview page, it is better that aligning Instruction with Overfitting. Also, codes like tag$li can used there to make instructions show like 1. 2. 3. I think that would be better.

2. In the reference page, i think it is better to add three lines br() before the license.

JuddHe committed:
change in overview page and reference page

align the "instruction" with "overfitting" in overview page.

change the tags ( dots to number 1,2,3) in overview page.

change the formatting of licensing in reference page

JuddHe commented:
formatting problems have been fixed.

On Thu, Jun 9, 2022 at 6:40 PM ymy5178 ***@***.***> wrote:

> ***@***.**** commented on this pull request.
>
> Recommended Priority: Urgent
>
> There are just two small format things.
>
>    1.
>
>    In the Overview page, it is better that aligning Instruction with
>    Overfitting. Also, codes like tag$li can used there to make instructions
>    show like 1. 2. 3. I think that would be better.
>    2.
>
>    In the reference page, i think it is better to add three lines br()
>    before the license.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/EducationShinyAppTeam/Overfitting/pull/3#pullrequestreview-1002077343>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AZFNJFYW7I2GQBCRBZAKBRTVOJXHJANCNFSM46RCV2BQ>
> .
> You are receiving this because your review was requested.Message ID:
> ***@***.***>
>

JuddHe committed:
change the reference page and description

JuddHe committed:
change reference page

ready to review

ymy5178 reviewed (dismissed):
I think the app is ready

JuddHe committed:
more fit style

JuddHe committed:
last update

JuddHe committed:
update readme

JuddHe committed:
readme

ymy5178 reviewed (commented):
I believe the app is ready.

JuddHe commented:
Hi, Phichchaya, can you help me to review the overfitting app again? thank
you!

Junjie

On Thu, Jun 16, 2022 at 9:22 PM Phichchaya27 ***@***.***>
wrote:

> ***@***.**** approved this pull request.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/EducationShinyAppTeam/Overfitting/pull/3#pullrequestreview-1009890632>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AZFNJF3ILTJ4FOZAWP3HXV3VPPHPFANCNFSM46RCV2BQ>
> .
> You are receiving this because your review was requested.Message ID:
> ***@***.***>
>

Phichchaya27 reviewed (dismissed):
I believe the app is ready!

ymy5178 reviewed (dismissed):
I believe the app is ready!

JuddHe committed:
check for citation in overview page

JuddHe committed:
icon change
